### PR TITLE
Idxopt

### PR DESCRIPTION
**Pull request by deadc0de6**
Add button on `Home` page for re-create search index

### CODE REVIEW

**Comment by mblayman** [comment]
This is convenient for an individual user, but I think it is not correct for a multi-user setup. A single user should not be able to re-index (because that's a potentially expensive operation that will block other Whoosh operations). It's reasonable to want a re-index link, however, I think this should go in the `administrate` template.

If you want to do that in this PR, go for it. Otherwise, we can close this PR and you can try again in a different PR.

**Comment by deadc0de6** [comment]
make sense ... I'll do it

File: markwiki/views/core.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def search():
 
 @app.route('/reindex/')
 def reindex():
+    '''force re-creation of index for search'''
```
Comment (mblayman) [inline markwiki/views/core.py]: I think docstrings should use English punctuation. In short:
```'''Force recreation of index for search.'''```

**Comment by mblayman** [comment]
One small docstring update then I'm good to merge. :+1: Thanks!

**Pull request merged**